apps/mobile-wallet/src/features/fund-password/fundPasswordStorage.ts

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Q: You still like the `features/***` way of ordering files? Do you see any clear pros and cons so far?

Reply:
I like it yes. Everything regarding the feature that is not being used in any other part of the app is contained within one folder.